lib/iris/fileformats/grib/__init__.py

```diff
@@ -47,80 +48,20 @@
 from iris.fileformats.grib._message import _GribMessage
 import iris.fileformats.grib.load_rules
 import iris.unit
+from iris.fileformats.grib._gribwrapper import (GribWrapper, grib_generator,
+                                                GribDataProxy)
 
-
-__all__ = ['grib_generator', 'load_cubes',
-           'reset_load_rules', 'save_grib2', 'GribWrapper',
-           'hindcast_workaround']
+__all__ = ['load_cubes', 'reset_load_rules', 'save_grib2']
```

Comment:
it was not my hope, but in the process of refactoring, I found it impossible to maintain this 'global' without resulting in a circular import situation. 

I'd be interested in an alternative, I have tried a number but nothing has been effective.

Perhaps an off-line conversation about managing imports could resolve this